docs/gitbook/architecture/topology/considerations.adoc

```diff
@@ -52,41 +52,43 @@ issues:
 - low transaction rate imposed by proof-of-work consensus
 - unified state managed by a single point in the network
 
-Proof-of-work consensus involves each node in the network performing
-computation-intensive calculations that take a notable amount of time to
-perform, but a negligible amount of time to verify. New blocks are added
-to the blockchain only after proof-of-work consensus has been reached,
-which limits the overall rate of adding to the blockchain considerably.
+Proof-of-work consensus (or _PoC_) involves each _mining_ node in the
+network performing computation-intensive calculations that take a
+notable amount of time to perform, but a negligible amount of time to
+verify. New blocks are added to the blockchain only after proof-of-work
+consensus has been reached, which limits the overall rate of adding to
+the blockchain considerably.
 
 For blockchains that use a single "master" node for ordering block
 creation, transactions complete more quickly than with proof of work,
 but all transaction processing is limited by the speed of the master
 node.
 
-Interbit blockchains use a consensus mechanism called
-link:https://dl.acm.org/citation.cfm?doid=571637.571640[Practical
-Byzantine Fault Tolerance] (PBFT), which can achieve concensus when no
-more than one third of the blockchain network's participating nodes
-disagree on a proposed change to the blockchain. Any node can propose a
-change -- dispatch an action -- and the network of nodes can very
-quickly reach consensus, often in less than a second. If a situation
-arises where consensus cannot be reached, the dispatched action is
-simply ignored. This architectural design solves both kinds of
-performance issues.
-
-PBFT consensus imposes some processing overhead, as proposed changes to
-the blockchain must propagate to the supporting nodes, the smart
+Interbit blockchains use a consensus algorithm called
+link:https://en.wikipedia.org/wiki/Proof-of-authority[Proof of
+Authority] (or _PoA_). When a new block is to be added to the
+blockchain, it contains a series of actions required to update the
+blockchain's state. The new block is then propagated to all of the other
+nodes, which act as _validators_. Each validator node has the authority
```

Comment:
These other nodes are not considered validators as they do not produce blocks and only receive them from the block master (sole "validator"). The one block master (named as validator in our config file) creates and propagates the block then the other nodes process the actions in the block. At the moment, no extra validation occurs.